src/api/api.js

```diff
@@ -0,0 +1,65 @@
+const fetch = require('node-fetch');
+const APIKey = process.env.VUE_APP_API_KEY;
+
+const getServiceResponse = async (city) => {
+    const APILink = `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${APIKey}`;
+
+    let response = await fetch(APILink);
+
+    if (response.ok) { 
+        let json = await response.json();
+        return json;
+    } else {
+       console.dir("HTTP-Error: " + response.status);
+    }
+}
+
+const dataHandler = ( city ) => {
```

Comment:
You drive me crazy `( city )`. It looks like somebody greedy wrote this or a person who has fear of confined spaces

Reply:
It would be better off to have `verb + something`. For instance, `getWeatherData`